packages/grafana-ui/src/components/FormField/FormField.tsx

```diff
@@ -41,7 +41,12 @@ export const FormField: FunctionComponent<Props> = ({
         {label}
       </InlineFormLabel>
       {inputEl || (
-        <input type="text" className={`gf-form-input ${inputWidth ? `width-${inputWidth}` : ''}`} {...inputProps} />
+        <input
+          type="text"
+          className={`gf-form-input ${inputWidth ? `width-${inputWidth}` : ''}`}
+          {...inputProps}
+          disabled={inputProps.disabled || inputProps.readOnly}
```

Comment:
As previous comment can't we continue to use `disabled` here?

Reply:
it doesn't always work

Reply:
fixed